playground/module-resolution/vite.config-no-prebundling.ts

```diff
@@ -0,0 +1,42 @@
+import { resolve } from 'node:path';
+import { cloudflare } from '@flarelabs-net/vite-plugin-cloudflare';
+import { defineConfig } from 'vite';
+
+export default defineConfig({
+	resolve: {
+		alias: {
+			'@alias/test': resolve(__dirname, './src/aliasing.ts'),
+		},
+	},
+	build: {
+		rollupOptions: {
+			// let's externalize this unconfigured alias just to make the build command pass
+			external: ['@alias/unconfigured'],
+		},
+	},
+	plugins: [
+		cloudflare({
+			workers: {
+				worker: {
+					main: './src/index.ts',
+					wranglerConfig: './src/wrangler.toml',
+					overrides: {
+						dev: {
+							optimizeDeps: {
+								exclude: [
+									'@cloudflare-dev-module-resolution/requires',
+									'react',
+								],
+							},
+						},
+						resolve: {
+							external: ['@cloudflare-dev-module-resolution/requires/ext'],
+						},
+					},
+				},
+			},
```

Comment:
Can you put a comment explaining what is going on here?

Reply:
sure 🙂👍

Reply:
I've added some comments, please have a look 🙏